libs/backend-api7/src/index.ts

```diff
@@ -8,16 +8,17 @@ import semver, { SemVer } from 'semver';
 import { Fetcher } from './fetcher';
 import { Operator } from './operator';
 import { ToADC } from './transformer';
+import * as typing from './typing';
 
 export class BackendAPI7 implements ADCSDK.Backend {
   private readonly client: AxiosInstance;
-  private readonly gatewayGroupName: string;
+  private readonly gatewayGroupName?: string;
```

Comment:
This is a misunderstanding. A `undefined/null` gatewayGroup is unacceptable for this backend.

Therefore, assuming it is undefined is inherently a logical error. If the gatewayGroup value passed in via opts could be null, it should be asserted there, and an error should be thrown if it is null.